BUILDING.md

```diff
@@ -112,6 +114,12 @@ From the grpc repository root
 ```sh
  $ make
 ```
+NOTE: if you get an error on linux such as 'aclocal-1.15: command not found', which can happen if you ran 'make' before installing the pre-reqs, try the following:
```

Comment:
What if we included "Common Problems" / "Build Troubleshooting" section at the end of the docs a listed a few most common build problems? The one you mentioned would be a good candidate to include (but it seems a bit random to include it here), another one is forgetting to update/fetch the submodules etc.